packages/cli/src/user-management/email/templates/password-reset-requested.mjml

```diff
@@ -0,0 +1,31 @@
+<mjml>
+	<mj-include path="./_common.mjml" />
+	<mj-body>
+		<mj-section>
+			<mj-column>
+				<mj-text font-size="24px" color="#ff6f5c">Reset your n8n password</mj-text>
+			</mj-column>
+		</mj-section>
+		<mj-section background-color="#FFFFFF" border="1px solid #ddd">
+			<mj-column>
+				<mj-text font-size="20px">Hi {{firstName}},</mj-text>
+				<mj-text>Somebody asked to reset your password on n8n ({{ domain }}).</mj-text>
```

Comment:
Just in case the appended dot might cause some email clients not to detect and render the URL:

```suggestion
				<mj-text>Somebody asked to reset your password on n8n ({{ domain }})</mj-text>
```